manifests/y/yetone/OpenAITranslator/0.0.36/yetone.OpenAITranslator.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created with WinGet Releaser v2 using Komac v1.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: yetone.OpenAITranslator
+PackageVersion: 0.0.36
+InstallerLocale: en-US
+InstallerType: wix
+ReleaseDate: 2023-03-28
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/yetone/openai-translator/releases/download/v0.0.36/OpenAI.Translator_0.0.36_x64_en-US.msi
+  InstallerSha256: 8AFA7176A12CBD8FEB4E749F2621BC7F46661536AB0F72CCB91C27FCBE65B8ED

```

Hi @yetone, the hash for this installer is incorrect.
```suggestion
  InstallerSha256: C49F82227F1469C86BEB8C61847F5AF29D4CE14F54E13DCE69F4F493C763AADB
```